extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -32,11 +32,31 @@ amp-sticky-ad {
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  overflow: visible !important;
+  direction: rtl;
```

Comment:
Reposting from above, but I think a more typical approach would be to do something like:
```
.myclas {
  right: 0;
}

[dir=rtl] .myclass {
  right: auto;
  left: 0;
}
```